lib/src/lib.rs

```diff
@@ -18,6 +18,8 @@ mod ctx;
 mod dbs;
 mod doc;
 mod err;
+#[cfg(feature = "parallel")]
+mod exe;
```

Comment:
One nit-pick, can we remove this from here, and put it into the `lib/src/exe/mod.rs` file?

```rs
#![cfg(feature = "parallel")]

use executor::{Executor, Task};
use once_cell::sync::Lazy;
use std::future::Future;
use std::panic::catch_unwind;
...
...
...
```

Reply:
I didn't even know this was possible, but I love it :star_struck: